cmake/std/atdm/ats2/environment.sh

```diff
@@ -69,105 +67,46 @@ else
   export ATDM_CONFIG_CTEST_PARALLEL_LEVEL=16
 fi
 
-# NOTE: We do *NOT* purge the modules first like on other systems because that
-# messes up the ATS-2 env.  Therefore, it is recommended that the user load a
-# new shell and then load one of these envs.
+# Purge then load StdEnv to get back to a fresh env in case previous other
+# modules were loaded.
+module purge --silent
+module load StdEnv
```

Comment:
This could be removed, StdEnv is sticky.

Reply:
Actually not.  Running `module purge --silent` blows bast sticky and removes all the modules!  Give it a try for yourself and see if you get the same behavior.